### PR TITLE
Feature/profile spec

### LINKED ISSUE #6: Header
As a user, when I navigate to any page in the app, the header will be viewable.

- [x] Title
- [x] Database nav button
- [x] Dashboard nav button

### LINKED ISSUE #13: Footer links/routes
[x] enable links/routes to portfolios
[x] enable link/route to employee docs for extension

### LINKED ISSUE #14: Cypress Test - Dash, Head, Foot, Incident Form

### PR DESCRIPTION
**Pull request by percworld**
### What Changed?
- Adds Header Component
- Adds Footer Component
- Adds Profiles Component
- Adds Passport Component
- Cypress testing for all User flows introduced 
### What's Next?
- Decide on some favorite svg's for use with error handling/ validation
### Known Issues?
- None
### Test Coverage?
- 100%
### README & Documentation
- Will add to Readme the github links associated with this update in the Profiles Component

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
-      <nav>
-        <NavLink to='/database' className='nav-link'>DATABASE</NavLink>
-        <NavLink to='/' className='nav-link'>INCIDENTS</NavLink>
-      </nav>
-      <Switch>
-        <Route exact path='/' component={IncidentsMenu} />
-        <Route path='/database' component={DatabaseMenu} />
-      </Switch>
+      <div className='content-wrap'>
+        <Header />
```
Comment (JeffShepherd) [inline src/components/App/App.js:16]: Solid refactor of the header into a standalone component

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.footer-container {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 2.5rem;  
+  display: flex;
+  justify-content: space-around;
+}
```
Comment (JeffShepherd) [inline src/components/Footer/Footer.css:8]: Let's think about making this super minimal and make it take up minimal space at the bottom of the page.

File: src/components/App/App.js (hunk 1 above)
Comment (cbmackintosh) [reply src/components/App/App.js:16]: Agreed. Always a good idea to break up the codebase into many components

File: src/components/Footer/Footer.css (hunk 2 above)
Comment (percworld) [reply src/components/Footer/Footer.css:8]: Sounds like a good idea

File: cypress/integration/quick-command-tests.js [hunk 3]
```diff
@@ -1,8 +1,72 @@
...
+        .contains('2021-05-21')
+    })
+  })
+
+  describe('DATABASE view', () => {
+    it('Shows a database of personnel', () => {
+      cy.get('a[data-cy=database-btn]').click()
+      cy.get('h2[data-cy=database-menu]').contains('Database Menu')
+    })
+  })
```
Comment (cbmackintosh) [inline cypress/integration/quick-command-tests.js:71]: Thanks for getting us started. We'll keep adding/updating as we add more features - good practice now that we're working with CI

**Review by cbmackintosh — APPROVED** [review]

**Review by JeffShepherd — APPROVED** [review]
Looks good!

**Pull request merged**